src/content/developers/docs/consensus-mechanisms/pos/pos-vs-pow/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Proof-of-stake vs proof-of-work
+description: A comparison between Ethereum's proof-of-stake and proof-of-work based consensus mechanism
+lang: en
+---
+
+Ethereum is a proof-of-stake blockchain. This has not always been the case. When Ethereum launched, proof-of-stake still needed a lot of research and development before it could be trusted to secure the real public network. Proof-of-work, however, was a simpler mechanism that had already been proven by Bitcoin, meaning it could be implemented right away to get the network off the ground. It took a further eight years to develop proof-of-stake to the point where it could be implemented.
+
+This page aims to explain the rationale behind Ethereum's switch to proof-of-stake from proof-of-work and explain the trade-offs.
+
+We also recommend watching this proof-of-work vs proof-of-stake [debate](https://www.youtube.com/watch?v=1m12zgJ42dI&t=3410s) from Bankless.
```

Comment:
I'd move this to further reading or [embed it at the end of the page](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/#visual-learner) personally. 

```suggestion
We also recommend watching this proof-of-work vs proof-of-stake [debate](https://www.youtube.com/watch?v=1m12zgJ42dI&t=3410s) from Bankless.
```